src/App.jsx

```diff
@@ -10,20 +11,26 @@ import Exchange from './components/Japan/Exchange';
 import Login from './components/Login/Login';
 
 function App() {
+  const [isLogIn, setIsLogIn] = useState(false);
+
+  // function handleLogButton() {
+  //   set
+  // }
+
   return (
     <BrowserRouter>
       <Routes>
         <Route path='/*' element={<Home />} />
-        <Route element={<KrDefaultLayout />}>
+        <Route element={<KrDefaultLayout isLogIn={isLogIn} onChangeIsLogIn={setIsLogIn} />}>
```

Comment:
isLogin은 로그인 컴포넌트인지? 로그인 시도 인지? 와 같은 혼동을 불러일으키므로 
isLoggedIn이 더 적절할 수 있어요. 
짧은 변수명보다 더 길지만 명확한 변수명이 좋다고 생각해요 !

In JavaScript, variable and function naming conventions typically follow the camelCase convention. Both "IsLogIn" and "isLoggedIn" are valid variable or function names, but "isLoggedIn" is the conventionally preferred choice in JavaScript.

Here's why "isLoggedIn" is generally better:

CamelCase Convention: JavaScript conventionally uses camelCase for variable and function names. CamelCase means that each word starts with a capital letter except the first one. "isLoggedIn" adheres to this convention, making your code more readable and consistent with established practices.

Readability: "isLoggedIn" is more readable because it clearly separates the words "is" and "logged" with a capital letter, making it easier to understand the purpose of the variable or function.

Consistency: Consistency in naming conventions is crucial for code maintainability and collaboration. Using "isLoggedIn" aligns with the common camelCase convention, which is used in many JavaScript projects and libraries.

**Avoids Confusion: "IsLogIn" might be interpreted as a boolean variable or function related to whether logging in is happening at the moment, while "isLoggedIn" clearly indicates that it's related to the current login status, which is usually represented as a boolean value.**

So, it's generally recommended to use "isLoggedIn" over "IsLogIn" in your JavaScript code to follow best practices and improve code readability.